### PR TITLE
IA-2567: Adding Tooltip next to the search field for org unit

### PR DESCRIPTION
**Pull request by butofleury**
Added Tooltip next to search field to explain possible options for searching an org unit!

Related JIRA tickets : [IA-2567](https://bluesquare.atlassian.net/browse/IA-2567) , [IA-2718](https://bluesquare.atlassian.net/browse/IA-2718)

## Self proofreading checklist

- [ ] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

Added Tooltip next to the search field for org unit

## How to test

Mouse over on the tooltip next the search field and see the appearing text.

## Print screen / video

OrgUnit-Tooltip Search

[IA-2567]: https://bluesquare.atlassian.net/browse/IA-2567?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[IA-2718]: https://bluesquare.atlassian.net/browse/IA-2718?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/app/translations/en.json:1]: can you please order the file by alphanumeric ?

File: hat/assets/js/apps/Iaso/domains/app/translations/fr.json
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/app/translations/fr.json:1]: same here

**Review by beygorghor — CHANGES_REQUESTED** [review]
Just ordering translations please

File: hat/assets/js/apps/Iaso/domains/app/translations/fr.json
Comment (butofleury) [reply hat/assets/js/apps/Iaso/domains/app/translations/fr.json:1]: Done with https://github.com/BLSQ/iaso/pull/1151/commits/e42a8c83105e2623a22b4bcf48cb349465eab9ff

Thanks!

**Review by beygorghor — APPROVED** [review]

**Pull request merged**